<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDeviceRestoreJobProgress</class>
 <widget class="QWidget" name="IDeviceRestoreJobProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="tSubtitleLabel" name="titleLabel">
     <property name="text">
      <string notr="true">DRIVE NAME</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string notr="true">TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="cancelButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-cancel">
       <normaloff>../../../../application/jobs/widgets</normaloff>../../../../application/jobs/widgets</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tSubtitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">tsubtitlelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
